service/api/client/client_test.go

```diff
@@ -172,16 +185,67 @@ func TestClient_AddRecord(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	t.Run("test add record", func(t *testing.T) {
-		rec, err := client.AddRecord(context.Background(), info.ID, body)
+	t.Run("test create record", func(t *testing.T) {
+		rec, err := client.CreateRecord(context.Background(), info.ID, body)
 		if err != nil {
-			t.Fatalf("failed to add record: %v", err)
+			t.Fatalf("failed to create record: %v", err)
 		}
 		if !rec.ThreadID().Equals(info.ID) {
-			t.Fatal("got bad thread ID from add record")
+			t.Fatal("got bad thread ID from create record")
 		}
 		if rec.LogID().String() == "" {
-			t.Fatal("got bad log ID from add record")
+			t.Fatal("got bad log ID from create record")
+		}
+	})
+}
+
+func TestClient_AddRecord(t *testing.T) {
```

Comment:
Here's a full AddRecord test. Only the client has the read key and the log private key.